parser/parser.y

```diff
@@ -257,6 +257,7 @@ import (
 	tinyIntType       "TINYINT"
 	tinytextType      "TINYTEXT"
 	to                "TO"
+	tokenRequire      "TOKEN_REQUIRE"
```

Comment:
Please make it unreserved. It may break compatibility if someone uses `token_require` as an identifier. E.g. `create table token_require(...) ...`.